test/test.js

```diff
@@ -95,6 +95,58 @@ describe('exiv2', function(){
       });
     });
   });
+  
+  describe('.setUtf8ImageTags()', function(){
```

Comment:
That's ok except it has to be to an Xmp property. Writing UTF-8 to Exif and IPTC wouldn't provoke the problem, even before the change.